composer.json

```diff
@@ -14,6 +14,7 @@
         "php": "^8.0.2",
         "ext-simplexml": "*",
         "barryvdh/laravel-ide-helper": "^2.13",
+        "fakerphp/faker": "^1.21",
```

Comment:
I couldn't find places where we use Faker on this package. Do you know them?

Reply:
https://github.com/psalm/psalm-plugin-laravel/blob/edc45c0cd193c3e82c3503154a56cb581e4b1e20/stubs/legacy-factories/FactoryBuilder.stubphp#L5